shopping/src/main/java/com/wfsample/shopping/ShoppingService.java

```diff
@@ -40,6 +42,7 @@
  * @author Srujan Narkedamalli (snarkedamall@wavefront.com).
  */
 public class ShoppingService extends Application<DropwizardServiceConfig> {
+  static Logger logger =  LogManager.getLogger(ShoppingService.class);
```

Comment:
Just trying to understand- In the ShoppingService class why we are logging with `logger.warn()`, whereas in other classes we log using `WavefrontTracer`?